jxl/src/image.rs

```diff
@@ -255,4 +255,56 @@ mod test {
         assert!(image.as_rect().to_pgm().starts_with(b"P5\n32 32\n255\n"));
         Ok(())
     }
+
+    #[cfg(feature = "debug_tools")]
```

Comment:
Might make sense to put these in the debug_tools module (in a further nested `test` submodule)

Reply:
Done